apps/src/maze/maze.js

```diff
@@ -58,18 +58,44 @@ const tiles = require('./tiles');
 
 module.exports = class Maze {
   constructor() {
+    this.scale = {
+      snapRadius: 1,
+      stepSpeed: 5
+    };
+
+    this.stepSpeed = 100;
+
+    this.cachedBlockStates = [];
+
     this.level;
     this.skin;
-    this.stepSpeed = 100;
+    this.startDirection;
+
+    this.map;
+    this.subtype;
+    this.executionInfo;
+
+    this.animating_;
+    this.response;
+    this.result;
+    this.testResults;
+    this.waitingForReport;
+
+    this.pegmanD;
+    this.pegmanX;
+    this.pegmanY;
+
+    this.MAZE_HEIGHT;
```

Comment:
Since these aren't actually constants, we may want to stop capitalizing them.

Reply:
Agreed; the goal of this PR is mostly to help draw the limits of what can and cannot be easily refactored, I'm gonna do most of the actual refactoring in other PRs